tests/bluetooth/tester/src/bttester.c

```diff
@@ -260,6 +261,57 @@ static uint8_t *recv_cb(uint8_t *buf, size_t *off)
 	return new_buf->data;
 }
 
+#if defined(CONFIG_UART_PIPE)
+/* Uart Pipe */
+static void uart_init(uint8_t *data)
+{
+	uart_pipe_register(data, BTP_MTU, recv_cb);
+}
+
+static void uart_send(uint8_t *data, size_t len)
+{
+	uart_pipe_send(data, len);
+}
+
+#else
+#define UART_DEV	"UART_0"
+static uint8_t *recv_buf;
+static size_t recv_off;
+static const struct device *dev;
+
+/* Uart Poll */
+static void uart_init(uint8_t *data)
+{
+	dev = device_get_binding(UART_DEV);
+	__ASSERT_NO_MSG((void *)dev);
+
+	recv_buf = data;
+
+	k_timer_start(&timer, K_MSEC(10), K_MSEC(10));
```

Comment:
The K_TIMER_DEFINE is below, doesn't that yield any compilers errrors?

Reply:
Fixed.